index.php

```diff
@@ -167,6 +167,11 @@ public function checkForUpdate() {
 		$versionString = isset($response['versionstring']) ? $response['versionstring'] : '';
 
 		if ($version !== '' && $version !== $this->currentVersion) {
+			if (PHP_INT_SIZE < 8 && Semver::satisfies($version, '> 25')) {
```

Comment:
why not version_compare?

Reply:
Good idea! Did not see that therr is already a function for that :)

Reply:
done